### PR TITLE
[xmltv] Added encoding option

### PR DESCRIPTION
**Pull request by radokristof**
For me the titles and other String type data will rendered incorrectly, because the encoding was not set correctly.
This commit adds the ability to change the encoding of the XMLTV file.

### CODE REVIEW

File: bundles/org.openhab.binding.xmltv/src/main/feature/feature.xml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<features name="org.openhab.binding.xmltv-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
-    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+<features name="org.openhab.binding.xmltv-${project.version}"
+	xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
```
Comment (Hilbrand) [inline bundles/org.openhab.binding.xmltv/src/main/feature/feature.xml]: Tags are preferred to be on a single line in feature files. Unfortunately spotless put the on a separate line. There is a fix for that, but not yet applied here. So could you put this features rag on a single line.

Comment (radokristof) [reply bundles/org.openhab.binding.xmltv/src/main/feature/feature.xml]: Yes, sure and this indeed was caused by the spotless formatter.

File: bundles/org.openhab.binding.xmltv/src/main/java/org/openhab/binding/xmltv/internal/handler/XmlTVHandler.java [hunk 2]
```diff
@@ -67,12 +69,11 @@ public void initialize() {
         logger.debug("Initializing {} for input file '{}'", getClass(), config.filePath);
 
         reloadJob = scheduler.scheduleWithFixedDelay(() -> {
-            final StreamSource source = new StreamSource(config.filePath);
             currentXmlFile = null;
             XMLStreamReader xsr = null;
             try {
                 // This can take some seconds depending upon weight of the XmlTV source file
-                xsr = xif.createXMLStreamReader(source);
+                xsr = xif.createXMLStreamReader(new FileInputStream(new File(config.filePath)), config.encoding);
```
Comment (J-N-K) [inline bundles/org.openhab.binding.xmltv/src/main/java/org/openhab/binding/xmltv/internal/handler/XmlTVHandler.java]: Is it really needed to have the encoding here? Using `FileInputStream` should return binary data and in my understanding this should be workling out of the box.

Comment (radokristof) [reply bundles/org.openhab.binding.xmltv/src/main/java/org/openhab/binding/xmltv/internal/handler/XmlTVHandler.java]: Unfortunately not and I don't know why.
Tried it without encoding, for example this text should be: "beszéd" and I got "beszĂŠd".

**Comment by radokristof** [comment]
@clinique I usually get the warning that there is not enough programmes in the XML file, think to refresh it, however I always update the XML every night for 3 days in advance... Why?
Looking at the code I don't find anything suspicious about this. Maybe XML processed wrongly?

**Pull request merged**